libstuff/sqlite3.c

```diff
@@ -455,7 +455,7 @@ extern "C" {
 */
```

Comment:
Are these changes that the sqlite guys made and you've just pulled into the repo?

Reply:
Yes, except the BedrockTester.cpp change to delete wal2 files.